validation/pipeline/tfp/Jenkinsfile.airgap.tests

```diff
@@ -0,0 +1,180 @@
+  def imageName = "tfp-automation-validation-${job_name}${env.BUILD_NUMBER}"
+  def testResultsOut = "results.xml"
+  def testResultsJSON = "results.json"
+  def envFile = ".env"
+  def config = env.CONFIG
+  def adminToken = ""
+  def privateRegistry = ""
+  def validationVolume = "ValidationSharedVolume-${job_name}${env.BUILD_NUMBER}"
+
+  def infraBranch = "${env.INFRA_BRANCH}"
+  if ("${env.INFRA_BRANCH}" != "null" && "${env.INFRA_BRANCH}" != "") {
+        infraBranch = "${env.INFRA_BRANCH}"
+  }
+  def testBranch = "${env.TEST_BRANCH}"
+  if ("${env.TEST_BRANCH}" != "null" && "${env.TEST_BRANCH}" != "") {
+        testBranch = "${env.TEST_BRANCH}"
+  }
+  def infraRepo = scm.userRemoteConfigs
+  if ("${env.INFRA_REPO}" != "null" && "${env.INFRA_REPO}" != "") {
+    infraRepo = [[url: "${env.INFRA_REPO}"]]
+  }
+  def testRepo = scm.userRemoteConfigs
+  if ("${env.TEST_REPO}" != "null" && "${env.TEST_REPO}" != "") {
+    testRepo = [[url: "${env.TEST_REPO}"]]
+  }
+  def timeout = "${env.TIMEOUT}"
+  if ("${env.TIMEOUT}" != "null" && "${env.TIMEOUT}" != "") {
+        timeout = "${env.TIMEOUT}" 
+  }
+
+    wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'XTerm', 'defaultFg': 2, 'defaultBg':1]) {
+        withFolderProperties {
+            paramsMap = []
+            params.each {
+                if (it.value && it.value.trim() != "") {
+                    paramsMap << "$it.key=$it.value"
+                }
+            }
+            withCredentials([ string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 'AWS_ACCESS_KEY_ID'),
+                              string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 'AWS_SECRET_ACCESS_KEY'),
+                              string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 'RANCHER_EKS_ACCESS_KEY'),
+                              string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 'RANCHER_EKS_SECRET_KEY'),
+                              string(credentialsId: 'AWS_SSH_PEM_KEY', variable: 'AWS_SSH_PEM_KEY'),
+                              string(credentialsId: 'RANCHER_SSH_KEY', variable: 'RANCHER_SSH_KEY'),
+                              string(credentialsId: 'AWS_SSH_RSA_KEY', variable: 'AWS_SSH_RSA_KEY'),
+                              string(credentialsId: 'AWS_SSH_KEY_NAME', variable: 'AWS_SSH_KEY_NAME'),
+                              string(credentialsId: 'RANCHER_REGISTRY_USER_NAME', variable: 'RANCHER_REGISTRY_USER_NAME'),
+                              string(credentialsId: 'RANCHER_REGISTRY_PASSWORD', variable: 'RANCHER_REGISTRY_PASSWORD'),
+                              string(credentialsId: 'ADMIN_PASSWORD', variable: 'ADMIN_PASSWORD'),
+                              string(credentialsId: 'USER_PASSWORD', variable: 'USER_PASSWORD'),
+                              string(credentialsId: 'RANCHER_VALID_TLS_CERT', variable: 'RANCHER_VALID_TLS_CERT'),
+                              string(credentialsId: 'RANCHER_VALID_TLS_KEY', variable: 'RANCHER_VALID_TLS_KEY')]) {
```

Comment:
lets remove stuff that isn't used

Reply:
Let me know what needs to be removed here